app/src/test/java/com/gzslt/imagesearch/main/imagedetails/mapper/ImageDetailsTupleToUiModelKtTest.kt

```diff
@@ -0,0 +1,160 @@
+package com.gzslt.imagesearch.main.imagedetails.mapper
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/zsoltGczy/image-search-android/blob/6ddf70a09e2c45836fdb95b5326f65131ccb690d/app/src/test/java/com/gzslt/imagesearch/main/imagedetails/mapper/ImageDetailsTupleToUiModelKtTest.kt#L1"></span>class ImageDetailsTupleToUiModelUnitTest should be declared in a file named ImageDetailsTupleToUiModelUnitTest.kt (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>